source/reference/operator/query/mod.txt

```diff
@@ -117,3 +117,31 @@ operator with an array that contains four elements:
    	"$err" : "bad query: BadValue malformed mod, too many elements",
    	"code" : 16810
    }
+
+Modulo with Decimal Divisors
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The ``modulo`` expression rounds down decimal input towards zero.
```

Comment:
We should to this expression as `$mod` (not Modulo). As written, it's reading like we're introducing a new concept because up until now, we've been referring to this expression exclusively as `$mod`.